src/components/RecordingFingerprint.tsx

```diff
@@ -34,6 +34,7 @@ const RecordingFingerprint = ({
     >
       {Object.keys(dataFeatures).map((k, idx) => (
         <ClickableTooltip
+          placement="auto"
```

Comment:
```suggestion
          placement="end-end"
```

Maybe experiment more to see if anything else works better?

Reply:
Have experimented with other placements and they have similar behaviours but none of them are obviously better than the one you have suggested